models/contacts.js

```diff
@@ -1,73 +1,61 @@
-const fs = require('fs');
-const path = require('path');
-const { v4: uuidv4 } = require('uuid');
-
-const contactsFilePath = path.join(__dirname, 'contacts.json');
-
-function readContactsFile() {
-  try {
-    const data = fs.readFileSync(contactsFilePath, 'utf8');
-    return JSON.parse(data);
-  } catch (error) {
-    console.error('Error reading contacts file:', error);
-    return [];
-  }
+const mongoose = require('mongoose');
+
+const contactSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: [true, 'Set name for contact'],
+  },
+  email: {
+    type: String,
+    required: true,
+  },
+  phone: {
+    type: String,
+    required: true,
+  },
+  favorite: {
+    type: Boolean,
+    default: false,
+  },
+});
+
+const Contact = mongoose.model('Contact', contactSchema);
+
+async function listContacts() {
```

Comment:
Staraj się pisać w jednakowym stylu, czyli że jeżeli używasz arrow function to to też dobrze zeby było pisane w tym samym stylu.